src/App.tsx

```diff
@@ -458,11 +459,16 @@ function App(): ReactElement {
     [replaceDataset]
   );
 
+  /** Attempts to update the current feature. Also updates the color ramp min/max if the feature has changed.
```

Comment:
Readability:  this comment.  What does "update" mean.  The function is already called updateFeature. Maybe it should be called attemptUpdateFeatureAndColorRamp ?  Info I would be looking for here:  why does it need an update? Update from what to what?  Like for example, it could say "this is called when we need to update an already loaded feature when a new dataset is loaded that contains the same feature."

Reply:
Ended up moving this into two separate methods, `replaceFeature` and `resetColorRampToDefaults`. I also had to move them higher up in `App.tsx`, because they were used before their declaration and it was causing another bug (!) in `replaceDataset`.